### PR TITLE
[MVC-1137]  Privacy policy newlang

### PR DESCRIPTION
**Pull request by nashthecoder**
[MVC-1137]  Privacy policy newlang

- Changed policy-lang posts to collections. Should allow for privacy policy to be created on siteleaf. 
-  Removed English from  https://triggerise.org/privacy-policy-english/
-  Redirects to the privacy policy page: https://triggerise.org/privacy-policy fixed. 

Screenshot 2021-06-28 at 21 33 05
Screenshot 2021-06-28 at 21 33 24

https://user-images.githubusercontent.com/15178823/123686504-8a10a380-d858-11eb-95ef-b77b2897908f.mov

[MVC-1137]: https://triggerise.atlassian.net/browse/MVC-1137

### CODE REVIEW

**Review by raguiar9080 — APPROVED** [review]
Looks great 👍 
Really cool seeing things be moved to collections 😄

**Review by franciscoovazevedo — APPROVED** [review]
Nice job!

File: _config.yml [hunk 1]
```diff
@@ -38,4 +35,22 @@ defaults:
...
+    type: privacy-policy
+  values:
+    label: Swahili
+    filter: Swahili
+- scope:
+    path: ''
+    type: privacy-policy
+  values:
+    label: French
+    filter: French
```
Comment (tiagovarela) [inline _config.yml]: Is this needed on `_config`? Would you need to add some entry here when adding a new lang to the collection?

Comment (nashthecoder) [reply _config.yml]: The initial one  - English was created when we tested it on Siteleaf and I git pulled from develop.

**Comment by nashthecoder** [comment]
> Looks great 👍
> Really cool seeing things be moved to collections 😄
> 
> 

Thanks to my filter guru @coertg for the helping hand ...

File: _config.yml (hunk 1 above)
Comment (tiagovarela) [reply _config.yml]: @nashthecoder shouldn't those fields be on the privacy-policy entry on the collection? (Also I'm not seeing the collection items in this PR are they missing?)

Comment (nashthecoder) [reply _config.yml]: @tiagovarela added the collections files.  Let me know if as, with the collections and config.yml it we are good to go on the merge.

Comment (tiagovarela) [reply _config.yml]: @nashthecoder Yes the collections look ok, although they probably don't need the layout field there. Could we remove the label and filter from the config.yml and just use the collection entry title field?

Comment (nashthecoder) [reply _config.yml]: @tiagovarela  - cleaned up the config.yml file.

**Review by nashthecoder — COMMENTED** [review]
@tiagovarela  this is done.

**Review by tiagovarela — APPROVED** [review]

**Pull request merged**